### PR TITLE
Fixed minor go convention violations and typos

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:

- [x] Fixed minor go convention violations and typos
- [x] Removed the test case `ScheduleBackupCreationSingleNS`

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by snigdha-px — APPROVED** [review]
LGTM

File: tests/backup/backup_restore_basic_test.go [hunk 1]
```diff
@@ -500,146 +500,6 @@ var _ = Describe("{DeleteAllBackupObjects}", func() {
 	})
 })
 
-// This testcase verifies schedule backup creation with a single namespace.
```
Comment (kphalgun-px) [inline tests/backup/backup_restore_basic_test.go:503]: Why removing this test case?

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/backup/backup_restore_basic_test.go (hunk 1 above)
Comment (mkoppal-px) [reply tests/backup/backup_restore_basic_test.go:503]: It was decided in our test case scrub meeting that this is already being covered in many other tests and hence it is redundant.

**Review by dbhatnagar-px — APPROVED** [review]
Lgtm

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**